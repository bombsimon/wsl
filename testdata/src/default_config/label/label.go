package testpkg

import "fmt"

func fn1() {
	for i := range make([]int, 2) {
		if i == 1 {
			goto END
		}

	END:
		_ = 1
	}
}

func fn2() {
	for i := range make([]int, 2) {
		if i == 1 {
			goto END
		}

	END:
	}
}

func fn3() {
L1:
	if true {
		_ = 1
	}
L2: // want `missing whitespace above this line \(never cuddle label\)`
	if true {
		_ = 1
	}

	_ = 1
L3: // want `missing whitespace above this line \(never cuddle label\)`
	_ = 1
}

func fn() {
LABEL:
	if true {
		fmt.Println("")
		fmt.Println("")
		fmt.Println("")
		fmt.Println("")
		break // want `missing whitespace above this line \(too many lines above branch\)`
	}
}
