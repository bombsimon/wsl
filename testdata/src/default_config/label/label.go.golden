package testpkg

import "fmt"

func fn1() {
	for i := range make([]int, 2) {
		if i == 1 {
			goto END
		}

	END:
		_ = 1
	}
}

func fn2() {
	for i := range make([]int, 2) {
		if i == 1 {
			goto END
		}

	END:
	}
}

func fn3() {
L1:
	if true {
		_ = 1
	}

L2: // want "missing whitespace decreases readability"
	if true {
		_ = 1
	}

	_ = 1

L3: // want "missing whitespace decreases readability"
	_ = 1
}

func fn() {
LABEL:
	if true {
		fmt.Println("")
		fmt.Println("")
		fmt.Println("")
		fmt.Println("")

		break // want "missing whitespace decreases readability"
	}
}
